ecosystem/sep-0008.md

```diff
@@ -126,6 +126,8 @@ Name | Data Type | Description
 status|string|`"action_required"`
 message|string|A human readable string containing information regarding the action required.
 action_url|string|A URL that allows the user to complete the actions required to have the transaction approved.
+action_params|string|A comma separated list of additional parameters that the client may optionally provide to the approval service; options are `first_name`, `last_name`, `email` and `public_key`
```

Comment:
Are these params query parameters or fields to be POSTed to the URL?

Given this list of params is likely to expand, and we might need to specify descriptions for each, can we start a table that lists the params and reference that table here?

Or, if this list is likely to overlap significantly with [SEP-9](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0009.md), I think we should reference that SEP and support any of those fields, using the field names there for consistency.

We could also probably transition towards using [SEP-12](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md) since that SEP provides greater control for a client to maintain and delete personal information, although that might be a breaking change and something we'd need to work towards.

Reply:
I thought about that as well, in our case it would be overhead. Yeah, the naming should be consistent with SEP-9; for us SEP-12 does not make sense as we have our own KYC Flow.

Reply:
Okay got it. If it works for you, let's change this to reference SEP-9 and say any parameters there may be requested as params. Also, let's refer to them as fields because that's what SEP-9 refers to them as.
```suggestion
action_fields|string| (Optional) A comma separated list of additional fields defined by [SEP-9](sep-0009.md) that the client may optionally provide to the approval service.
```